# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Controllers ---------------------------------------
joint1_position_controller:
  type: effort_controllers/JointPositionController
  joint: base_link__link_01
  pid: {p: 100.0, i: 15, d: 0.0}
joint2_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_01__link_02
  pid: {p: 1000.0, i: 50, d: 3.0}
joint3_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_02__link_03
  pid: {p: 1000.0, i: 50, d: 3.0}
joint4_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_03__link_04
  pid: {p: 100.0, i: 50, d: 0.0}

# Gripper Controllers ----------------------------------------
joint5_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_04__grip_l
  pid: {p: 1050.0, i: 90, d: 0.0}
joint6_position_controller:
  type: effort_controllers/JointPositionController
  joint: link_04__grip_r
  pid: {p: 1050.0, i: 90, d: 0.0}
